src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Sami32 It doesn't surprise me if your CPU is quiet, there's almost nothing happening there. I guess it is from back in the days of passive CPU cooling 😛 

I can't see any difference between the two commands you posted, why post it twice? Also, you don't seem to get it, but there's no support for ```_cuvid``` in the nightly I downloaded:
```powershell
Unknown decoder 'h265_cuvid'
```